### PR TITLE
OSDOCS-619: Adding content for schedulable master nodes

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@mburke5678 Would you mind doing the peer review for this, since it's adding content in your area? Especially feedback on the placement, I'm not sure if this location works or if there is a better place for it. Thanks!

**Comment by bergerhoffer** [comment]
@ravisantoshgudimetla Can you please review this and let me know if you have any feedback? I will also get this mentioned in the release notes.

**Comment by bergerhoffer** [comment]
I just noticed that the assignee was changed on https://jira.coreos.com/browse/WRKLDS-5. So @ravisantoshgudimetla or @damemi - could one of your review this update please?

Preview: https://osdocs-619--ocpdocs.netlify.com/openshift-enterprise/latest/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-master-schedulable_nodes-nodes-working

File: modules/nodes-nodes-working-master-schedulable.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-working.adoc
+
+[id="nodes-nodes-working-master-schedulable_{context}"]
+= Configuring master nodes as schedulable
+
+You can configure master nodes to be schedulable, meaning that new Pods are allowed for placement on the master nodes. By default, master nodes are not schedulable. However, if your cluster does not contain any worker nodes, then master nodes are marked schedulable by default.
```
Comment (ravisantoshgudimetla) [inline modules/nodes-nodes-working-master-schedulable.adoc]: I think, we should be explicit that from 4.2, we support doing so

**Review by ravisantoshgudimetla — COMMENTED** [review]
Mostly LGTM, except for some nits.

Thanks @bergerhoffer for the PR

File: modules/nodes-nodes-working-master-schedulable.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/nodes-nodes-working-master-schedulable.adoc]: Thanks, I'll add this.

File: modules/nodes-nodes-working-master-schedulable.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+= Configuring master nodes as schedulable
+
+As of {product-title} 4.2, you can configure master nodes to be
+schedulable, meaning that new Pods are allowed for placement on the master
+nodes. By default, master nodes are not schedulable. However, if your cluster
+does not contain any worker nodes, then master nodes are marked schedulable by
+default.
+
+You can allow or disallow master nodes to be schedulable by configuring the
+`mastersSchedulable` field.
```
Comment (mburke5678) [inline modules/nodes-nodes-working-master-schedulable.adoc]: @bergerhoffer @ravisantoshgudimetla Do we need a note here: _If your cluster does not contain any worker nodes, do not configure the masters as unschedulable?_ 
(I'm assuming in such a cluster there are no other nodes than master.)

**Comment by mburke5678** [comment]
@bergerhoffer One question. Otherwise LGTM

File: modules/nodes-nodes-working-master-schedulable.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/nodes-nodes-working-master-schedulable.adoc]: Thanks for the question @mburke5678, I don't know the answer to that. @ravisantoshgudimetla - let me know if something like this is worthwhile to add.

I'm going to go ahead and merge this, but we can always add it in a new PR if needed.

**Pull request merged**